tests/scattering1d/test_numpy_scattering1d.py

```diff
@@ -114,6 +114,16 @@ def test_Scattering1D_average_global(self, backend):
         Sx = sc(x)
         assert Sx.shape[-1] == 1
 
+    def test_981(self, backend):
```

Comment:
can you give this a more indicative name? it's a test that should be covered any way

Reply:
it's quite customary to name tests after issue numbers when those issues were raised after finding a bug.
this is why i'm setting `T=2` here